satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -145,60 +152,15 @@ read one (
 	where  node.id = ?
 )
 
-// "Find" query; returns nil if node not found
-read scalar (
-	select node
-	where  node.id = ?
-)
-
 read all (
 	select node.id
 )
 
-//--- overlaycache ---//
-
-model overlay_cache_node (
-	key    node_id
-	unique node_id
-
-	field node_id   blob
-	field node_type int
-
-	field address   text (updatable) // TODO: use compressed format
-	field protocol  int  (updatable)
-	
-	field operator_email  text (updatable)
-	field operator_wallet text (updatable) //TODO: use compressed format
-	
-	field free_bandwidth int64 (updatable)
-	field free_disk      int64 (updatable)
-
-	field latency_90           int64  (updatable)
-	
-	field audit_success_ratio  float64 (updatable)
-	field audit_uptime_ratio   float64 (updatable)
-	field audit_count          int64   (updatable)
-	field audit_success_count  int64   (updatable)
-
-	field uptime_count         int64 (updatable)
-	field uptime_success_count int64 (updatable)
-)
-
-create overlay_cache_node ( )
-
-read one (
-	select overlay_cache_node
-	where  overlay_cache_node.node_id = ?
-)
-
 read limitoffset (
-	select overlay_cache_node
-	where  overlay_cache_node.node_id >= ?
+	select node
+	where  node.id >= ?
```

Comment:
Should we add an order by?

Reply:
No idea. I am not changing any logic in this PR. What would be the benefit of using order by?

Reply:
https://www.postgresql.org/docs/8.3/queries-limit.html

"The query optimizer takes LIMIT into account ... you are very likely to get different plans (yielding different row orders) depending on what you give for LIMIT and OFFSET. Thus ... inconsistent results unless you enforce a predictable result ordering with ORDER BY. This is not a bug"

Reply:
OK. I see. Good point. Please check my latest commit if I've done it right.

Reply:
looks good!